build/android/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<string name="preparing_media">Preparing media...</string>
+	<string name="preparing_media">Preparing media…</string>
```

Comment:
Please don't use this special 'row of dots' character, we use separate full stops everywhere and it's clearer.